src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -379,6 +382,22 @@ const Stream = React.createClass({
     return (<ul className="group-list" ref="groupList">{groupNodes}</ul>);
   },
 
+  renderAwaitingEvents() {
+    let org = this.getOrganization();
+    let project = this.getProject();
+
+    return (
+      <div className="box awaiting-events">
+        <div className="wrap">
+          <div className="robot"></div>
+          <h3>Waiting for events…</h3>
+          <p>{t('Our error robot is waiting to')} <span className="strikethrough">{t('devour')}</span> {t('recieve your first event.')}</p>
```

Comment:
Thanks - this was the piece I was primarily concerned with. Pushing the change now.